meson.build

```
@@ -0,0 +1,46 @@
+project(
+  'fabtris2',
+  'c',
+  default_options : ['c_std=c11']
```

```suggestion
  default_options : ['c_std=gnu11']
```
Some non-vanilla features are used, so gnu11 is required.